conftest.py

```diff
@@ -99,3 +120,24 @@ def job_2(user_HR):
                          marked_count=0,
                          apply_link=DEFAULT_APPLY_LINK)
     return job
+
+
+@pytest.fixture
+def valid_job_data(user_HR):
+    job_data = {'publisher': user_HR,
+                'company_name': DEFAULT_COMPANY_NAME,
+                'title': 'Software Engineer',
+                'location': DEFAULT_JOB_LOCATION,
+                'description': 'Part time software engineer at Wix: Back end',
+                'post_until': '2022-12-23',
+                'is_active': True,
+                'apply_link': DEFAULT_APPLY_LINK}
+    return job_data
```

Comment:
Why wont you use `Job` class instead?

Reply:
Following our discussion offline about #78 (https://github.com/redhat-beyond/Side-Hustle/pull/78#discussion_r776848574), please try to add a class method to Job that converts it to dict.